app/scripts/controllers/client/ViewClientController.js

```diff
@@ -12,6 +12,85 @@
             scope.openShares = true ;
             scope.updateDefaultSavings = false;
             scope.charges = [];
+
+            // address
+            scope.addresses=[];
+            scope.view={};
+            var entityname="ADDRESS";
+            scope.addressTypeId={};
+            formdata={};
+
+
+            resourceFactory.clientTemplateResource.get(function(data)
+            {
+                scope.enableAddress=data.isAddressEnabled;
+                if(scope.enableAddress===true)
+                {
+
+                    resourceFactory.addressFieldConfiguration.get({entity:entityname},function(data){
+                        scope.view.addresstypId=data[0].is_enabled;
```

Comment:
Usually, we should not have hard coded indexes like this. Problem with this implementation is in future somebody adds a new type in between these indexes, that person need to comeback and look into these existing indexes also